R/loo_subsample.R

```diff
@@ -34,7 +34,9 @@ loo_subsample <- function(x, ...) {
 #' @template function
 #' @param data,draws,... For `loo_subsample.function()`, these are the data,
 #'   posterior draws, and other arguments to pass to the log-likelihood
-#'   function.
+#'   function. Note that for some `loo_approximation`s, the draws will be replaced
+#'   by the posteriors summary statistics to compute loo approximations. See
+#'   `loo_approximation` for details.
```

Comment:
Did you mean `loo_approximate_posterior` instead of `loo_approximation`? If so you can use `[loo_approximate_posterior()]` in the doc and it will auto link to that doc page.

Reply:
Actually not. I was refering to the argument loo_approximation in the same function. Can that be clarified more somhow?

Reply:
Oh I just forgot that was the argument name. How about "See argument `loo_approximation` for details."?

Reply:
Now fixed.